web/frontend/src/components/camera/stream-manager.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  type Client,
+  StreamClient,
+} from '@viamrobotics/sdk';
+import { CameraManager } from './camera-manager';
+
+export class StreamManager {
+  streamClient: StreamClient;
+
+  constructor (
+    private client:Client,
+    public cameraManagers: Map<string, CameraManager> = new Map<string, CameraManager>()
+
+  ) {
+    this.streamClient = new StreamClient(client);
+  }
+
+  setCameraManager (cameraName:string) {
+    const tempManager = new CameraManager(cameraName, this.client, this.streamClient);
```

Comment:
To go with the above changes.

```suggestion
    const tempManager = new CameraManager(this.client, cameraName, this.streamClient);
```